saleor/graphql/product/types/products.py

```diff
@@ -168,8 +173,14 @@ class Meta:
         description = "Represents availability of a product in the storefront."
 
 
-@key(fields="id")
+@key(fields="id channelSlug")
 class ProductVariant(ChannelContextTypeWithMetadata, CountableDjangoObjectType):
+    channel_slug = graphene.String(
```

Comment:
We should rename all newly introduced `channel_slug` fields to just `channel` - some time ago we did this unification in the schema and we should stay consistent with it.